README.md

```diff
@@ -183,6 +183,9 @@ the following form (`[...]` denotes optional or conditional values),
  :fetcher [git|github|gitlab|bitbucket|bzr|hg|darcs|fossil|svn|cvs|wiki]
  [:url "<repo url>"]
  [:repo "github-gitlab-or-bitbucket-user/repo-name"]
+ [:commit "<ref>"]
+ [:branch "<branch-name>"]
+ [:version "<regexp>"]
```

Comment:
i think maybe we should pick something like `:version-regexp` or `:versionregexp` as a name, because `:version` makes me fear we're going to get some PRs where people will put in `2.5.4` or something.